### ISSUE TITLE
No color in function, for, end, struct

### ISSUE THREAD

**Issue by gobenavides** [wg030/jlcode#14]
First things first: what an awesome package you have created! Lots of thanks.

Now, my problem:

I'm having trouble displaying the file found in https://github.com/gobenavides/DunklZernikeExpansions/blob/master/src/DunklZernikeExpansions.jl correctly.
More specifically, "function", "for", "end" and "struct" are not appearing boldly. 

Since I have plenty of special unicode characters I have to add them manually in the preamble, which I attach entirely in order to be able to track any conflict.

```
\documentclass[12pt, oneside]{book}
\usepackage[utf8]{inputenc}
\textwidth 17cm
\textheight 22.5cm
\oddsidemargin 0.25cm
\addtolength{\voffset}{-1.5cm}
\addtolength{\hoffset}{-0.5cm}
\usepackage{ragged2e}
\usepackage{fancyhdr}
\usepackage[Sonny]{fncychap}

\ChTitleVar{\LARGE\bf}


\setlength{\parindent}{12pt}
\setlength{\parskip}{3pt}
\usepackage{amssymb,amsmath,epsfig}
\usepackage{amsthm}
\usepackage{thmtools}
\usepackage{thm-restate}
\usepackage[colorlinks=false,breaklinks=true,linkcolor=blue]{hyperref}
\usepackage{graphics,psfrag,graphicx,color}
\usepackage{float,hhline}
\usepackage{rotating}
\usepackage{diagbox}
\usepackage{setspace}
\usepackage{bibentry}
\nobibliography*

\usepackage{xcolor}
\newcommand{\rednote}[1]{{\color[hsb]{.0,1.0,.75}#1}}
\newenvironment{redarea}{\rednote\bgroup}{\egroup}
\newcommand{\violetnote}[1]{{\color[hsb]{.833,1.0,.55}#1}}
\newenvironment{violetarea}{\violetnote\bgroup}{\egroup}


\usepackage[charsperline=100,usecolors=true]{jlcode}
\addlitjlbase{÷}{\textdiv}{1}
\addlitjlbase{γ1}{$\gamma 1$}{2}
\addlitjlbase{γ2}{$\gamma 2$}{2}
\addlitjlbase{κ1}{$\kappa 1$}{2}
\addlitjlbase{κ2}{$\kappa 2$}{2}
\addlitjlbase{Dunkl–Zernike}{Dunkl--Zernike}{13}
```
I'm loading my file via `\lstinputlisting[breaklines=true,numbers=left]{Julia-code/DunklZernikeExpansions.jl}`

I would really appreciate if you help me fixing this issue.

**Comment by wg030** [comment]
Hey gobenavides,
nice to hear that you like the jlcode package.

I have looked at you problem and it seems as if this is not a problem of the package, but a problem with the font you have choosen (probably the default computer modern font?!).
The problem is that the package displays code with ttfamily, i.e the typewriter font, and keywords are displayed with bfseries. Now, if your choosen font does not have support for a bold typewriter style, it will simply just display a bold tyewriter text as normal typewriter text.

Long story short, to fix the problem, just change your font to a font that supports bold typewriter.
Alternatively, just change the typewriter family to courier with the following command in your preamble:
`\renewcommand{\ttdefault}{pcr}`

I guess that this should solve your problem.

**Comment by gobenavides** [comment]
You're right, the font was the problem. Maybe it would be a good idea to add this observation in the manual.

Thanks!

**Comment by wg030** [comment]
> You're right, the font was the problem. Maybe it would be a good idea to add this observation to the manual.

Sure! I will add a hint to the manual when releasing the next version!
